src/mozilla_sec_eia/models/sec10k/notebooks/exhibit21_extractor.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        "layoutlm_training_run": "layoutlm-labeledv0.2",
</pre><p>Does setting a different value in the Dagster config overrule what's set in this cell?</p><p> </p><p>What actually does this string represent? Is it the model path from MLflow, something from Dagster, or something local? I see&nbsp;<code>exhibit21_extractor</code>&nbsp;and&nbsp;<code>layoutlm_extractor</code>&nbsp;on the MLFlow registered models, but neither seems right.</p><p> </p><p>Should this be&nbsp;<code>layoutlm_training_run</code>&nbsp;or&nbsp;<code>layoutlm_uri</code>? The documentation cell above says&nbsp;<code>layoutlm_uri</code>&nbsp;but maybe just needs to be updated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/mozilla-sec-eia/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>